src/cloud-api-adaptor/test/e2e/common_suite.go

```diff
@@ -151,7 +151,8 @@ func DoTestCreatePeerPodContainerWithExternalIPAccess(t *testing.T, e env.Enviro
 
 func DoTestCreatePeerPodWithJob(t *testing.T, e env.Environment, assert CloudAssert) {
 	jobName := "job-pi"
-	job := NewJob(E2eNamespace, jobName)
+	image := "quay.io/prometheus/busybox:latest"
+	job := NewJob(E2eNamespace, image, 8, jobName)
```

Comment:
Ordering of arguments is wrong here. Noticed it due to e2e failure
method signature: NewJob(namespace, name string, backoffLimit int32, image string, command ...string)